Resources/config/hallo_verden_http_exceptions.yaml

```diff
@@ -0,0 +1,3 @@
+hallo_verden_http_exceptions:
+  exception_log:
+    use_flatten_exception: false
```

Comment:
Defaulting this to false will cause a breaking change, no?

Reply:
The actual default is set in the Configuration class. but yeah, this file should reflect that. good catch. fixed in 6864a57